### ISSUE TITLE
"Today" badge maybe?

### ISSUE THREAD

**Issue by valignatev** [tsoding/schedule#77]
IMO it's not immediately obvious which stream is today or closest. I found "starts in" sign, but it was an effort. How about adding a bright "today" or "next" badge?

**Comment by rexim** [comment]
@valignatev yeah I also find it difficult to read sometimes. Thanks for the issue! We will implement that.

**Comment by rexim** [comment]
@valignatev after #96 when there are no events for the day we have a "Day off" panel. So now the top non-grey event is the event for today. Does it make it easier to read?
